functions/system.bash

```diff
@@ -11,6 +11,15 @@ whiptail_check() {
 
 system_upgrade() {
   echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
+  # TODO: Remove updating the key on each update after some time (in 2020)
+  cond_redirect wget -O openhab-key.asc 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab'
+  cond_redirect apt-key add openhab-key.asc
+  if [ $? -ne 0 ]; then 
+    echo "FAILED (updating OH key)"
+    echo -n "attempting to continue..."
+  else
+    rm -f openhab-key.asc
```

Comment:
Unconditional?

Reply:
Do you mean we shouldn't need the else? I can create another PR if so?

Reply:
Yes feels like this can potentially end in a file written to the current folder. We should in any case always try to use `/tmp` files 🤔
Don't bother, it's a temporary fix and should be okay.